### PR TITLE
On-Chain Primary Address

### PR DESCRIPTION
**Pull request by prxt6529**

### CODE REVIEW

File: components/delegation/DelegationCenterMenu.tsx [hunk 1]
```diff
@@ -19,12 +19,14 @@ import NewSubDelegationComponent from "./NewSubDelegation";
...
 import WalletCheckerComponent from "./walletChecker/WalletChecker";
 import { useRouter } from "next/router";
+import NewAssignPrimaryAddress from "./NewAssignPrimaryAddress";
 
 export enum DelegationCenterSection {
   CENTER = "delegation-center",
   REGISTER_DELEGATION = "register-delegation",
   REGISTER_SUB_DELEGATION = "register-sub-delegation",
   REGISTER_CONSOLIDATION = "register-consolidation",
+  ASSING_PRIMARY_ADDRESS = "assign-primary-address",
```
Comment (brookr) [inline components/delegation/DelegationCenterMenu.tsx:29]: @prxt6529 should this be `ASSIGN_PRIMARY_ADDRESS`?

File: components/delegation/DelegationCenterMenu.tsx [hunk 2]
```diff
@@ -134,6 +136,24 @@ export default function DelegationCenterMenu(props: Readonly<Props>) {
             }}
           />
         );
+      case DelegationCenterSection.ASSING_PRIMARY_ADDRESS:
```
Comment (brookr) [inline components/delegation/DelegationCenterMenu.tsx:139]: @prxt6529 should this be `ASSIGN_PRIMARY_ADDRESS`?

File: components/delegation/DelegationCenterMenu.tsx (hunk 1 above)
Comment (prxt6529) [reply components/delegation/DelegationCenterMenu.tsx]: sure should be!

Comment (prxt6529) [reply components/delegation/DelegationCenterMenu.tsx]: updated

**Review by brookr — APPROVED** [review]
LGTM!

**Pull request merged**